.github/workflows/publish-charmed-bind-edge.yaml

```diff
@@ -0,0 +1,28 @@
+name: Publish charmed-bind to edge
+
+on:
+  push:
+    branches:
+      - main
+      - track/*
+    paths:
+      - 'charmed-bind/**'
+
+jobs:
+  publish-to-edge-charmed-bind:
+    name: Build snap
+    uses: canonical/data-platform-workflows/.github/workflows/build_snap.yaml@v30.1.3
+    with:
+      path-to-snap-project-directory: ./charmed-bind
+
+    name: Release snap
+    needs:
+      - build
+    uses: canonical/data-platform-workflows/.github/workflows/release_snap.yaml@v30.1.3
+    with:
+      channel: latest/edge
+      artifact-prefix: ${{ needs.build.outputs.artifact-prefix }}
+    secrets:
+      snap-store-token: ${{ secrets.CHARMED_BIND_EDGE_SNAPCRAFT_TOKEN }}
```

Comment:
Do you actually need different tokens for each snap? From charmcraft, you can use the same for any charm

Reply:
No you're right. I tested locally and it works with the same token. Fixed